src/Spinners/LoadSpinner.tsx

```diff
@@ -0,0 +1,43 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSpinner } from '@fortawesome/free-solid-svg-icons';
+import { BaseTheme } from '../Theme';
+
+
+export interface LoadSpinnerProps {
+  /** Allows you to pass in style properties for the element */
+  theme?: object;
+  /** Specifies the text under the spinner */
+  children: ReactElement;
+}
+
+const StyledLoadSpinner = styled.div`
+  background: transparent;
+  color: ${({ theme }): string => (theme.color.background.dark)};
+  &:hover {
+    color: ${({ theme }): string => (theme.color.background.darker)};
+  }
+  font: ${({ theme }): string => (theme.font.heading)};
+  display: inline-block;
```

Comment:
I don't think `inline-block` is correct here -- it should still be a `block` element.

I would also prefer to use [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout) instead of `text-align` here, as noted in the suggested change above. They're almost doing the same thing, but flexbox is meant for handling more complex layouts like this, where `text-align` is specifically for handling the display of text.